### PR TITLE
chore(roll): roll Playwright to 1.26.0-alpha-sep-1-2022

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright-dotnet/issues/2272

### CODE REVIEW

File: src/Playwright.NUnit/PlaywrightTestAttribute.cs [hunk 1]
```diff
@@ -100,6 +100,10 @@ private static string GenerateOutputLog(List<TestResult> failedResults, int retr
                     output += $"\nFailing test runs:\n";
                     foreach (var result in failedResults)
                     {
+                        if (result == null)
```
Comment (mxschmitt) [inline src/Playwright.NUnit/PlaywrightTestAttribute.cs:103]: drive-by linting fix, since result and result.Message could be null.

File: src/Playwright.Tests/PageGotoTests.cs [hunk 2]
```diff
@@ -583,10 +582,11 @@ public async Task ShouldReturnWhenNavigationIsComittedIfCommitIsSpecified()
                 // Write enought bytes of the body to trigge response received event.
                 var str = "<title>" + new string('a', 4100);
                 await context.Response.WriteAsync(str);
-                await context.Response.BodyWriter.FlushAsync();
+                await context.Response.Body.FlushAsync();
+                await Task.Delay(5_000);
```
Comment (yury-s) [inline src/Playwright.Tests/PageGotoTests.cs:586]: does it have to be so long?

**Review by yury-s — APPROVED** [review]
